src/api/lib/db.coffee

```diff
@@ -1,26 +1,20 @@
-#
-# Wraps & exports a mongojs instance. Automatically selects collections based
-# on the folder names under /apps. https://github.com/mafintosh/mongojs
-#
-
-mongojs = require 'mongojs'
+{ MongoClient } = require 'mongodb'
 fs = require 'fs'
 path = require 'path'
 { MONGOHQ_URL } = process.env
 debug = require('debug') 'api'
 
-collections = ['articles', 'users', 'sections', 'artists', 'curations', 'channels', 'tags', 'verticals', 'authors', 'sessions']
-db = mongojs MONGOHQ_URL, collections
```

Comment:
This `collections` array appears to be necessary for the previous library `mongojs`. It seems fine to remove.